src/Sinch/Verification/VerificationMethod.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Sinch.Core;
+
+namespace Sinch.Verification
+{
+    /// <summary>
+    ///     The method of the verification request.
+    /// </summary>
+    /// <param name="Value"></param>
+    public record VerificationMethod(string Value) : EnumRecord(Value)
```

Comment:
Btw, is there a reason why this one is a `record` whereas the `VerificationTypeInternal` (which could be renamed to `VerificationMethod`) is a `class` ?

Reply:
Internal is for use of as constants in code. Record is specifically for this param: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-status/#tag/Verification-status/operation/VerificationStatusByIdentity!in=path&path=method&t=request